css/theme-lava.css

```diff
@@ -1341,9 +1341,19 @@ nav .container {
   -webkit-transition: all 0.3s ease-out;
   -moz-transition: all 0.3s ease-out;
 }
+/* .speaker .image-holder {
```

Comment:
No need to add another block of code 
You can insert your changes in the same block